DOCS/new-license-workflow.md

```diff
@@ -1,8 +1,15 @@
-This document provides guidance and checklists for SPDX legal team members who are assigned to shepherd a new license request.  This instructions here assume the requestor has already submitted the new license/exception request via the [SPDX online tools](http://13.57.134.254/app/).
+# New License Workflow
 
-New license requests are to be assigned to an SPDX legal team member and that person is responsible for following the request through to its final determination.  Make sure to add yourself as the Assignee, add the label “new license/exception request”, and the appropriate release milestone to the issue.
+> ⚠️ **NOTA BENE:** A GitHub account is required for the New License Workflow. If you do not have a GitHub account yet, you must [create one](https://github.com/join) (for free) to continue with the workflow.
+
+This document provides guidance and checklists for SPDX legal team members who are assigned to shepherd a new license request.  
```

Comment:
I suppose the two spaces at the end of the line were not intentional, right?

```suggestion
This document provides guidance and checklists for SPDX legal team members who are assigned to shepherd a new license request.
```

Reply:
I'll be adding a commit with a couple of other tweaks, and will include this.